packages/plugin-core/src/web/commands/lookup/NoteLookupProvider.ts

```diff
@@ -45,7 +45,7 @@ export class NoteLookupProvider implements ILookupProvider {
     try {
       // if empty string, show all 1st level results
       if (transformedQuery.queryString === "") {
-        const items = this.fetchRootQuickPickResults({
+        const items = await this.fetchRootQuickPickResults({
```

Comment:
this await is actually redundant since items is getting returned immediately. This can be condensed to `return this.fetchRootQuickPickResults()`

Separate note: can we replace this impl with `fetchRootResultsFromEngine` that you added in the lookup utils?